packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -135,6 +138,8 @@ export function InitializeBindings<SpecificBindings extends AnyBindings>({
 
     component.componentWillLoad = function () {
       const element = getElement(this);
+      element.setAttribute(renderedAttribute, 'false');
```

Comment:
I use the element to store information has it appeared doing `component['something']` was only saved once per component type

Reply:
Is this something we could have caught with a test ?